From 03b7c7dc952cba9d0f127b13ba7cfa8ef93b4f13 Mon Sep 17 00:00:00 2001
From: Robert Chiras <robert.chiras@nxp.com>
Date: Thu, 9 Nov 2017 11:33:31 +0200
Subject: [PATCH 2861/5242] MLK-16918-6: phy: update Mixel mipi-dsi phy driver
 for MX8MQ

commit  f7f9cd98b07c6839cf71f89eb23bdab2b97b0411 from
https://source.codeaurora.org/external/imx/linux-imx.git

Added compatible string for i.MX8MQ platform.

Signed-off-by: Robert Chiras <robert.chiras@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 drivers/phy/phy-mixel-mipi-dsi.c |    2 ++
 1 file changed, 2 insertions(+)

diff --git a/drivers/phy/phy-mixel-mipi-dsi.c b/drivers/phy/phy-mixel-mipi-dsi.c
index 934456e..cd7192d 100644
--- a/drivers/phy/phy-mixel-mipi-dsi.c
+++ b/drivers/phy/phy-mixel-mipi-dsi.c
@@ -345,10 +345,12 @@ static int mixel_mipi_phy_power_off(struct phy *phy)
 
 static struct devtype imx8qm_dev = { .have_sc = true };
 static struct devtype imx8qxp_dev = { .have_sc = true };
+static struct devtype imx8mq_dev = { .have_sc = false };
 
 static const struct of_device_id mixel_mipi_phy_of_match[] = {
 	{ .compatible = "mixel,imx8qm-mipi-dsi-phy", .data = &imx8qm_dev },
 	{ .compatible = "mixel,imx8qxp-mipi-dsi-phy", .data = &imx8qxp_dev },
+	{ .compatible = "mixel,imx8mq-mipi-dsi-phy", .data = &imx8mq_dev },
 	{}
 };
 MODULE_DEVICE_TABLE(of, mixel_mipi_phy_of_match);
-- 
1.7.9.5

